### DISCUSSION TITLE
How to delete a contact

### DISCUSSION THREAD

**Discussion by HoffmannP** [nmshd/feedback#25] [General]
Is there any way in the app do remove a contact? At least for testing purposes I'd like to be able to remove and add a contact multiple times to test our onboarding procedure.

**Comment by tnotheis** [comment]
There is currently no option to delete relationships. However, we are actively working on this possibility. You can expect a solution in the next few weeks.
Due to some uncertainties caused by an internal reorganization, we are unfortunately unable to give you a more precise date.

**Comment by HoffmannP** [comment]
For anyone how want's to develop an app integrating the onboarding process: You need to delete your profile so you can restart the process.
